solidity/contracts/ConnextVestingWallet.sol

```diff
@@ -0,0 +1,101 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.20;
+
+import {VestingWallet, VestingWalletWithCliff} from './VestingWalletWithCliff.sol';
+
+import {Ownable} from '@openzeppelin/contracts/access/Ownable.sol';
+import {Ownable2Step} from '@openzeppelin/contracts/access/Ownable2Step.sol';
+import {IERC20} from '@openzeppelin/contracts/token/ERC20/IERC20.sol';
+
+/**
+ * @dev Vesting schedule:
+ *      - 1/13 of the tokens will be released after 1 year, starting from _initTimestamp
+ *      - 1/13 of the tokens will be released every month after that, for 12 months
+ *
+ *     The equivalent vesting schedule has a 13 months duration, with a 1 month cliff, offsetted to
+ *     start from e.g. Jan 1st 2023: At Feb 1st 2023 the cliff is triggered unlocking
+ *     1/13 of the tokens, and then 1/13 of the tokens will be linearly unlocked every month after that.
+ */
+contract ConnextVestingWallet is VestingWalletWithCliff, Ownable2Step {
+  uint64 public constant ONE_YEAR = 365 days;
+  uint64 public constant ONE_MONTH = ONE_YEAR / 12;
+  address public constant NEXT_TOKEN = 0xFE67A4450907459c3e1FFf623aA927dD4e28c67a; // Mainnet NEXT token
+
+  uint64 public constant VESTING_OFFSET = ONE_YEAR - ONE_MONTH;
+  uint64 public constant VESTING_DURATION = ONE_YEAR + ONE_MONTH;
+  uint64 public constant VESTING_CLIFF_DURATION = ONE_MONTH;
+
+  uint64 public initTimestamp;
+  uint256 public totalAmount;
+
+  constructor(
+    uint64 _initTimestamp,
+    address _beneficiary,
+    uint256 _totalAmount
+  ) VestingWalletWithCliff(_beneficiary, _initTimestamp + VESTING_OFFSET, VESTING_DURATION, VESTING_CLIFF_DURATION) {
```

Comment:
@wei3erHase instead of `_initTimestamp + VESTING_OFFSET` used just `_initTimestamp` iirc. Imo it's wrong, double check it please